### PR TITLE
Linkt markdown and rst using travis.

### PR DESCRIPTION
**Pull request by gwarf**
Linkt markdown and rst using travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+install:
+  - gem install mdl
+  - pip install rstcheck
+  - pip install sphinx
+script:
+  # Linting
+  # https://github.com/markdownlint/markdownlint
+  - mdl -s relaxed README.md
+  # https://github.com/myint/rstcheck
+  - rstcheck --report warning source/*.rst
```
Comment (enolfc) [inline .travis.yml]: not brave enough to put `--report info` :)

**Comment by enolfc** [comment]
Do we really need codacy for documentation?

**Comment by gwarf** [comment]
No idea why codacy is here in fact, I didn't had it AFAIK. But yes, I think it's useless (and that's why I didn't made it required)

File: .travis.yml (hunk 1 above)
Comment (gwarf) [reply .travis.yml]: The fact is that it was complaining about missing usage of references that where in fact existing in different files. :)

**Pull request merged**